src/connectors/hierarchical-menu/connectHierarchicalMenu.js

```diff
@@ -96,7 +96,7 @@ export default function connectHierarchicalMenu(renderFn, unmountFn = noop) {
     const {
       toggleShowMore,
       setToggleShowMore,
-      isShowingMore,
+      getIsShowingMore,
```

Comment:
What do you think of `checkIsShowingMore`?
I always struggle with naming boolean variable and function returning boolean.
Not sure what's the best.

Reply:
it could maybe also be a getter on the helper, so it can stay `isShowingMore` instead of a function